microk8s-resources/actions/common/utils.sh

```diff
@@ -323,26 +323,35 @@ produce_certs() {
         fi
     done
 
+    render_ca_conf
+    if ! [ -f "${SNAP_DATA}/certs/ca.conf" ]; then
+        echo "changeme" >  "${SNAP_DATA}/certs/ca.conf"
+    fi
+
+    if ! "${SNAP}/usr/bin/cmp" -s "${SNAP_DATA}/certs/ca.conf.rendered" "${SNAP_DATA}/certs/ca.conf"; then
+        cp ${SNAP_DATA}/certs/ca.conf.rendered ${SNAP_DATA}/certs/ca.conf
```

Comment:
This does not look right. I think that if there is a change in the ca we should set the force flag on this function so all related certificates are recreated.